### PR TITLE
Issue/1933 Create "Migrating" section and move content from Guides

### LINKED ISSUE #1933: [Dependency: Guides] - Create "Migrating" section and move content from Guides
Dependency for #1859 

Goal: As a user I would like to be able to easily find (starting from the homepage), how to upgrade to the latest version of webpack.

- [ ] Create a new Section called "Migrate", it's creation should be at the same tier as "Guides"
  - [ ] The url to access this page should be `webpack.js.org/migrate`
- [ ] Move existing Migrating (v1 => v2/3) Guide and place it inside of new `Migrating` section under 
  - [ ] This page's url should be `webpack.js.org/migrate/3`

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Create "Migrating" section and move content from Guides.
Fixes #1933

### CODE REVIEW

File: src/content/migrate/3.md [hunk 1]
```diff
@@ -14,14 +14,13 @@ contributors:
...
   - selbekk
   - ndelangen
+  - EugeneHlushko
 ---
 
 The following sections describe the major changes from webpack 1 to 2.
 
 T> Note that there were far fewer changes between 2 and 3, so that migration shouldn't be too bad. If you are running into issues, please see [the changelog](https://github.com/webpack/webpack/releases) for details.
 
-W> This content may be moved to the blog post in the near future as version 2 has been out for a while. On top of that, version 3 was recently released and version 4 is on the horizon. As noted above, folks should instead to refer to [the changelog](https://github.com/webpack/webpack/releases) for migrations.
```
Comment (EugeneHlushko) [inline src/content/migrate/3.md]: I dont think this is relevant anymore

Comment (EugeneHlushko) [reply src/content/migrate/3.md]: Also previous note also leading to /wp/releases

Comment (TheLarkInn) [reply src/content/migrate/3.md]: Agreed

**Comment by EugeneHlushko** [comment]
Thank you, it is now `master` based @TheLarkInn

File: src/content/migrate/3.md (hunk 1 above)
Comment (dhruvdutt) [inline src/content/migrate/3.md]: Take a look at changes done in [#1939](https://github.com/webpack/webpack.js.org/pull/1939/files#diff-91aa65efa4b8d708cd748987e4397532R49).

**Comment by EugeneHlushko** [comment]
@TheLarkInn can you re-approve this one been hanging for some time now

**Comment by EugeneHlushko** [comment]
Hi @montogeek ,
Can u merge this if i rebase and resolve the conflict?

**Comment by montogeek** [comment]
@EugeneHlushko sure

**Comment by EugeneHlushko** [comment]
Failure is known: API rate limit exceeded for 104.154.120.187

Pls finish this one @montogeek

**Comment by EugeneHlushko** [comment]
@montogeek yes, makes any sense for the finished PRs? thought to create new ones under main repository but merge older ones as is to reduce mess on my side of things

**Comment by montogeek** [comment]
Was trying to make the build pass, it already did. Already pinged @TheLarkInn to approve it

**Comment by EugeneHlushko** [comment]
Thank you!

File: antwar.config.js [hunk 2]
```diff
@@ -55,7 +55,8 @@ module.exports = {
         'code-splitting-css': '/guides/code-splitting',
         'code-splitting-libraries': '/guides/code-splitting',
         'why-webpack': '/comparison',
-        'production-build': '/guides/production'
+        'production-build': '/guides/production',
+				'migrating': '/migrate/3',
```
Comment (ogonkov) [inline antwar.config.js]: Indent

**Comment by TheLarkInn** [comment]
Approved. Let's get conflicts resolved and good to go!

**Comment by EugeneHlushko** [comment]
Cool, waiting for build and can merge cc @montogeek

**Comment by montogeek** [comment]
Thanks

**Pull request merged**